nipype/interfaces/base.py

```diff
@@ -1134,8 +1134,9 @@ def run(self, **inputs):
 
                     runtime.prof_dict = {
                         'time': vals[:, 0].tolist(),
-                        'mem_gb': (vals[:, 1] / 1024).tolist(),
+                        'rss_gb': (vals[:, 1] / 1024).tolist(),
                         'cpus': vals[:, 2].tolist(),
+                        'vms_gb': (vals[:, 3] / 1024).tolist(),
```

Comment:
just a nitpick here - should we make the `gb` be `gib` or `GiB` to maintain consistency with the term Gibibytes.

Reply:
I guess I'd either prefer `GiB` (since `gib` does not say "gibibyte" to me) or just to switch to decimal factors and keep variable names as `gb`. I'm not sure whether changing the variable name or the log base would be least disruptive, so I'll leave that to your call @oesteban @satra.

Reply:
i think both are disruptive. since more people will use the name, i think changing the name ensures that people using it accounts for it. changing the log base implies that people using it may interpret the data incorrectly. it's possible this is already happening.

my vote would be for changing the name.

Reply:
I don't think there are a critical mass of people to maintain a name that poorly represents the concept. Let's change it to `GiB` before more people start using the monitor, following @effigies preference :+1: .